daemon/src/taker_cfd_actor.rs

```diff
@@ -253,3 +287,88 @@ fn setup_contract(
 
     (actor, sender)
 }
+
+fn verify_cets(
+    oracle_pk: &schnorrsig::PublicKey,
+    maker: &PartyParams,
+    taker_cets: &[(
+        Transaction,
+        EcdsaAdaptorSignature,
+        Vec<u8>,
+        schnorrsig::PublicKey,
+    )],
+    msg1: &Msg1,
```

Comment:
:upside_down_face: I would just pass in the `Vec<(Txid, EcdsaAdaptorSignature)>`.